assets/app/components/site/editor/editorContainer.js

```diff
@@ -22,7 +22,7 @@ const propTypes = {
 };
 
 const redirectToFileOnBranch = (siteId, branch, filePath) => {
-  routeActions.redirect(`/sites/${siteId}/edit/${branch}/${filePath}`);
+  routeActions.redirect(`/sites/${siteId}/edit/${branch}/${filePath}`, 'replace');
```

Comment:
Oh! I should have looked closer. Github had both commits labeled as "7 days ago". These look great. Merging.